### ISSUE TITLE
is Transfer-Encoding supported for blob uploads?

### ISSUE THREAD

**Issue by CheyenneForbes** [crate/crate#6991]

**Comment by joemoe** [comment]
hey @CheyenneForbes, no this is not support right now. which type of Transfer-Encoding would you use? thanks, joe

**Comment by CheyenneForbes** [comment]
chunked... it would help a lot with my project and the large files I add to crate

**Comment by joemoe** [comment]
ok, thanks for suggesting. we might get back to that.

**Comment by CheyenneForbes** [comment]
could it be added during the next week? I'd really like to launch the platform before march ends

**Comment by joemoe** [comment]
no, we won't be able to this in the next week.
i'm sorry to say this is not very high on our priority list at the moment as this is the first request for that, so i recommend you don't rely on that feature with your platform.

**Comment by joemoe** [comment]
hey @CheyenneForbes, 

we thought about that and implemented it. we also backported it to all our releases back to the very first cratedb release, supported and unsupported.

no just joking, we did some tests to see if we are passing the parameters through in the right way and actually found out that due to our netty integration, we are already supporting this. so sorry for the wrong information. that should also work with our python driver.

here is an example:

```
$ sha1sum big-file.iso

$ curl --verbose -X PUT -H "Transfer-Encoding: chunked" http://localhost:4200/_blobs/t1/<SHA1SUM> -T ./big-file.iso
```

happy to see your platform online. we'd love to get more insights.
